test-data/unit/check-incremental.test

```diff
@@ -4474,8 +4476,8 @@ def outer() -> None:
 [out]
 [out2]
 
-[case testRecursiveAliasImported]
-# flags: --no-new-semantic-analyzer
+[case testRecursiveAliasImported-skip]
```

Comment:
It looks like this test case can be removed, as the next case covers is roughly the same for the new semantic analyzer. Here the behavior in the new analyzer is different by design.